lib/phobos_db_checkpoint/cli.rb

```diff
@@ -34,23 +34,35 @@ def init
              aliases: ['-d'],
              default: 'db/migrate',
              banner: 'Destination folder relative to your project'
+      option :config,
+             aliases: ['-c'],
+             default: 'config/database.yml',
+             banner: 'Database configuration relative to your project'
       def copy_migrations
+        if options[:config]
+          ENV['DB_CONFIG'] = options[:config]
+        end
+
+        unless (ActiveRecord::Base.connection_pool.with_connection { |con| con.active? } rescue false)
+          PhobosDBCheckpoint.configure(pool_size: 1)
+        end
+
         destination_fullpath = File.join(destination_root, options[:destination])
         generated_migrations = list_migrations(destination_fullpath)
         FileUtils.mkdir_p(destination_fullpath)
-
+        file_path = nil
         template_migrations_metadata.each do |metadata|
-
           if migration_exists?(generated_migrations, metadata[:name])
             say_status('exists', metadata[:name])
-
           else
             file_path = File.join(options[:destination], "#{metadata[:number]}_#{metadata[:name]}")
             template_path = File.join('templates/migrate', metadata[:path])
             template(template_path, file_path)
           end
-
         end
+      rescue
+        file_path && FileUtils.rm(file_path)
```

Comment:
Is it possible that we enter the rescue block, with the `file_path` not nil, but the file was not created yet?
If so, `rm` will raise another error, hiding the original failure